watertap/property_models/seawater_prop_pack.py

```diff
@@ -707,7 +707,180 @@ def build(self):
+            initialize=-1.443e-1,
+            units=pyunits.J / pyunits.kg / pyunits.K**3,
+            doc="Entropy parameter B6",
+        )
+        self.entr_mass_param_B7 = Var(
+            within=Reals,
+            initialize=5.879e-4,
+            units=pyunits.J / pyunits.kg / pyunits.K**4,
+            doc="Entropy parameter B7",
+        )
+        self.entr_mass_param_B8 = Var(
+            within=Reals,
+            initialize=-6.111e1,
+            units=pyunits.J / pyunits.kg / pyunits.K**2,
+            doc="Entropy parameter B8",
+        )
+        self.entr_mass_param_B9 = Var(
+            within=Reals,
+            initialize=8.041e1,
+            units=pyunits.J / pyunits.kg / pyunits.K**2,
+            doc="Entropy parameter B9",
+        )
+        self.entr_mass_param_B10 = Var(
+            within=Reals,
+            initialize=-3.035e-1,
+            units=pyunits.J / pyunits.kg / pyunits.K**3,
+            doc="Entropy parameter B10",
+        )
+        self.entr_mass_param_C1 = Var(
+            within=Reals,
+            initialize=0.1543,
+            units=pyunits.J / pyunits.kg / pyunits.K,
+            doc="Entropy pure water parameter C1",
+        )
+        self.entr_mass_param_C2 = Var(
+            within=Reals,
+            initialize=15.383,
+            units=pyunits.J / pyunits.kg / pyunits.K**2,
+            doc="Entropy pure water parameter C2",
+        )
+        self.entr_mass_param_C3 = Var(
+            within=Reals,
+            initialize=-2.996e-2,
+            units=pyunits.J / pyunits.kg / pyunits.K**3,
+            doc="Entropy pure water parameter C3",
+        )
+        self.entr_mass_param_C4 = Var(
+            within=Reals,
+            initialize=8.193e-5,
+            units=pyunits.J / pyunits.kg / pyunits.K**4,
+            doc="Entropy pure water parameter C4",
+        )
+        self.entr_mass_param_C5 = Var(
+            within=Reals,
+            initialize=-1.370e-7,
+            units=pyunits.J / pyunits.kg / pyunits.K**5,
+            doc="Entropy pure water parameter C5",
+        )
+        self.gibbs_mass_param_B1 = Var(
+            within=Reals,
+            initialize=-2.4176e2,
+            units=pyunits.J / pyunits.kg / pyunits.K,
+            doc="Gibbs parameter B1",
+        )
+        self.gibbs_mass_param_B2 = Var(
+            within=Reals,
+            initialize=-6.2462e-1,
+            units=pyunits.J / pyunits.kg / pyunits.K**2,
+            doc="Gibbs parameter B2",
+        )
+        self.gibbs_mass_param_B3 = Var(
+            within=Reals,
+            initialize=7.4761e-3,
+            units=pyunits.J / pyunits.kg / pyunits.K**3,
+            doc="Gibbs parameter B3",
+        )
+        self.gibbs_mass_param_B4 = Var(
+            within=Reals,
+            initialize=1.3836e-3,
+            units=pyunits.J / pyunits.kg / pyunits.K**2,
+            doc="Gibbs parameter B4",
+        )
+        self.gibbs_mass_param_B5 = Var(
```

Comment:
I think some tests failed due to unit inconsistency attributed to gibbs parameters and implementation in associated constraint (I may be mixing this up with another property, but my point is you'll need to resolve unit inconsistency nevertheless)